packages/shorebird_cli/test/src/commands/release/ios_releaser_test.dart

```diff
@@ -282,6 +284,54 @@ void main() {
           ).thenAnswer((_) async => flutterVersionAndRevision);
         });
 
+        group('when a patch signing key path is provided', () {
+          late File patchSigningPublicKeyFile;
+
+          setUp(() {
+            patchSigningPublicKeyFile = File(
+              p.join(
+                Directory.systemTemp.createTempSync().path,
+                'patch-signing-public-key.pem',
```

Comment:
This doesn't affect the logic of the tests, but public keys are consumed in a `.der` format